js/src/utils/tracks.js

```diff
@@ -24,13 +24,40 @@ export const recordStepContinueEvent = noop;
  * @return {Object} Event properties with base event properties.
  */
 export function addBaseEventProperties( eventProperties ) {
-	const { slug } = sfwData;
-	const { version } = select( STORE_KEY ).getGeneral();
+	const { trackingSlug } = sfwData;
+	const { version, adAccountId } = select( STORE_KEY ).getSettings();
 
 	const mixedProperties = {
 		...eventProperties,
-		[ `${ slug }_version` ]: version,
+		[ `${ trackingSlug }version` ]: version || sfwData.pluginVersion,
```

Comment:
@Sidsector9 I think it's clearer if we remove the `_` from the `trackingSlug/slug` variable and instead have it here:
```suggestion
		[ `${ trackingSlug }_version` ]: version || sfwData.pluginVersion,
```

Reply:
If we are hydrating the `general` property of the store with the version, then we won't need the `sfwData.pluginVersion` fallback.